tools/make-source

```diff
@@ -26,26 +26,9 @@ fi
 base=$(cd $(dirname $0); pwd -P)
 outdir=$(pwd -P)
 
-# Make a .tar.gz with the source code
-
 cd $base/..
+./autogen.sh 1>&2
+make --silent dist-gzip V=1 distdir=cockpit-wip 1>&2
```

Comment:
If out of tree build directory is used then the cockpit-wip.tar.gz file ends up in an unexpected place.